app/dialog/otioproperties/otiopropertiesdialog.cpp

```diff
@@ -0,0 +1,80 @@
+/***
+  Olive - Non-Linear Video Editor
+  Copyright (C) 2019  Olive Team
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+***/
+
+#include "otiopropertiesdialog.h"
+
+#include <QDialogButtonBox>
+#include <QFileDialog>
+#include <QFileInfo>
+#include <QLabel>
+#include <QPushButton>
+#include <QVBoxLayout>
+
+#include "core.h"
+#include "dialog/sequence/sequence.h"
+
+namespace olive {
+
+OTIOPropertiesDialog::OTIOPropertiesDialog(const QList<Sequence*>& sequences, Project* active_project, QWidget* parent)
+    :
+    QDialog(parent),
+    sequences_(sequences)
+{
+  QVBoxLayout* layout = new QVBoxLayout(this);
+
+  layout->addWidget(new QLabel(tr("Change settings for each OTIO sequence to be loaded")));
```

Comment:
Prefer using "OpenTimelineIO" in user-facing strings rather than "OTIO". Also perhaps this could be a little more descriptive, e.g. mentioning OTIO files don't contain sequence parameters so please enter the correct parameters for each sequence here.